server/svix-server/src/core/message_app.rs

```diff
@@ -181,11 +181,11 @@ impl TryFrom<endpoint::Model> for CreateMessageEndpoint {
     }
 }
 
-kv_def!(AppEndpointKey, CreateMessageApp);
+cache_kv_def!(AppEndpointKey, CreateMessageApp, "SVIX_CACHE_APP_v3");
 impl AppEndpointKey {
     // FIXME: Rewrite doc comment when AppEndpointValue members are known
     /// Returns a key for fetching all cached endpoints for a given organization and application.
     pub fn new(org: OrganizationId, app: ApplicationId) -> AppEndpointKey {
-        AppEndpointKey(format!("{}_APP_v3_{}_{}", Self::PREFIX_CACHE, org, app))
+        AppEndpointKey(format!("_APP_v3_{}_{}", org, app))
```

Comment:
This looks wrong? Shouldn't it be:
```suggestion
        AppEndpointKey(format!("{}_{}", org, app))
```
And I'm not sure where the _ between the prefix and the key go.

I gotta say, I'm not sure I'm a huge fan of having the prefix defined separately than the function, it makes it very hard to know what's going on. It's much easier to just see:
```rust
         AppEndpointKey(format!("SVIX_CACHE_APP_v3_{}_{}", org, app))
```
All in one place.